app/javascript/packages/document-capture/components/document-capture-warning.tsx

```diff
@@ -39,14 +41,29 @@ function DocumentCaptureWarning({
   const subHeading = !nonIppOrFailedResult && !isFailedDocType && (
     <h2>{t('errors.doc_auth.rate_limited_subheading')}</h2>
   );
+  const subHeadingRef = useRef<HTMLDivElement>(null);
+  const errorTextRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    const subHeadingText = subHeadingRef.current?.textContent;
+    const errorText = errorTextRef.current?.textContent;
+
+    trackEvent('IdV: warning shown', {
+      location: 'doc_auth_review_issues',
+      remaining_attempts: remainingAttempts,
+      heading,
+      subHeading: subHeadingText,
```

Comment:
Can we choose a consistent case for all these properties, re: snake_casing in `remaining_attempts` and camelCasing in `subHeading` ? No strong preference for me and the conventions are a little inconsistent, though since `remaining_attempts` already existed here, maybe we go with snake_case for internal consistency?

i.e.

```suggestion
      remaining_attempts: remainingAttempts,
      heading,
      sub_heading: subHeadingText,
```

Alternatively, "subheading" is a single word, so it probably shouldn't be cased at all.

```suggestion
      remaining_attempts: remainingAttempts,
      heading,
      subheading: subheadingText,
```

Reply:
These are good points about consistency.

Re: `subheading` vs `subHeading`, I was following the pattern established in the app but I will timebox fixing it to one word everywhere (in the docauth realm, anyway) and see how much of a lift it is.

re: `snake_case` vs `camelCase`, I checked the app as a whole for times we use `trackEvent` and it does look like we're pretty inconsistent on which case we use. I'll stick with snake_case (but as you pointed out, fixing `subheading` will fix the consistency anyway).

Reply:
It turned out to be a quick fix! These issues were (mostly) fixed in 814d669bed831fc21f24da1470191c17584cc89e.

Also adding a note that I changed the error message name (and finished fixing the above camel vs snake case issue) from `errorText` to `error_message_displayed` in 5a7e69bc21d6b6e28299bfdef7e6bf9ae7d20c86 based on product management feedback.